src/cli/run.rs

```diff
@@ -81,18 +104,24 @@ pub async fn execute(args: Args) -> miette::Result<()> {
                 eprintln!();
             }
             eprintln!(
-                "{}{}",
-                console::style("✨ Pixi task: ").bold(),
+                "{}{}{}{}{}",
+                console::Emoji("✨ ", ""),
+                console::style("Pixi task (").bold(),
+                console::style(executable_task.run_environment.name())
```

Comment:
I think it would be nice to change this message to say:
```
✨ Pixi task: `install` in `default` environment: 
        cargo install --path . --locked
```
As we can now also reach the name of the task. And possibly only show the `(default)` if we have environments defined.